bin/generate-docs.sh

```diff
@@ -85,11 +86,13 @@ function generate_scanner_docs() {
       "docs/README.DockerHub-Scanner.md" \
       "${scanner_dir}/.helm-docs.gotmpl" \
       "${HELM_DOCS_DIR}/README.DockerHub-Scanner.md.gotmpl"
+      # XXX: #754 Why is here no generation for ArtifactHub?
```

Comment:
Similar to the parsers, these are for "custom build scanner docker images", meaning projects like nmap which don't have a official prebuild docker image, where we have to build and publish our own.